iaso/tests/models/test_storage.py

```diff
@@ -0,0 +1,19 @@
+from iaso.models import Account, StorageDevice
+from iaso.test import TestCase
+
+
+class StorageModelTestCase(TestCase):
+    @classmethod
+    def setUpTestData(cls):
+        cls.account = Account.objects.create(name="test")
+
+    def test_new_storages_have_correct_status(self):
+        """New storages have the OK status, and decent values for the related"""
+        storage = StorageDevice.objects.create(account=self.account, type="NFC", customer_chosen_id="123")
+
+        self.assertEqual(storage.status, StorageDevice.OK)
+        self.assertEqual(storage.status_reason, "")
+        self.assertEqual(storage.status_comment, "")
+        self.assertEqual(
+            storage.status_updated_at.replace(microsecond=0), storage.created_at.replace(microsecond=0)
```

Comment:
I'm convinced that this will fail once in a while when we're  at 0,999999999 microsecond for the first timestamp and 1,0000000001 for the second (while the time difference is still acceptable). Would be better to make a difference here

Reply:
Indeed, will fix that ASAP. Thanks!